app/assets/stylesheets/main.scss

```diff
@@ -131,34 +134,23 @@ u {
   padding-bottom: 10px;
 }
 
-.stripe:not(.reverse):first-child {
-  padding: 80px 0px 40px;
+.stripe {
+  padding: 20px 0;
 }
```

Comment:
How come you removed these paddings?

For me the screenshots you've shown feel a little too squashed together now.

Reply:
I removed these for mobile view, but I'll tweak them cause I agree

Reply:
OK I have tweaked the spacing, and the type a bit more, you can see how that looks in the updated screenshots in the original comment on this PR.